assets/js/buttons-examples.js

```diff
@@ -524,6 +524,10 @@ $(document).ready(function () {
           classes: "discord",
           text: "Login with Discord",
         },
+        {
```

Comment:
Please place this in the correct alphabetical order under "Social"

Reply:
sure I'll re-arrange it.

Reply:
Issue Resolved.